src/flowops_library.c

```diff
@@ -172,8 +172,13 @@ flowop_connect(strand_t *sp, flowop_t *fp)
 	datap->p_id = fp->p_id;
 
 	error = datap->connect(datap, &fp->options);
-	if (error == UPERF_SUCCESS)
+	if (error == UPERF_SUCCESS) {
```

Comment:
Looks like a tabs vs spaces formatting problem.

Reply:
thanks for catching that. Fixed it.